NSS-Day-Cohort-55/tabloidfullstack-adequately-powerful-ibexes
# Wm create post

# Description
Allows user to create a new post
Closes Ticket #42


# Testing Instructions
Upon login, select "Add Post"
Fill out the form with information for a post
Click Add Post when Finished
You will be re-routed to a view of the current logged in users posts.
Check your DB to confirm the post is there and the ImageLocation table is NOT null
